service/src/main/kotlin/io/provenance/digitalcurrency/consortium/frameworks/AddressTagQueue.kt

```diff
@@ -48,10 +48,25 @@ class AddressTagQueue(
         transaction {
             AddressRegistrationRecord.findForUpdate(message.id).first().let { addressRegistration ->
                 withMdc(*addressRegistration.mdc()) {
-                    when (addressRegistration.status) {
-                        INSERTED -> addressTagService.createEvent(addressRegistration)
-                        PENDING_TAG -> addressTagService.eventComplete(addressRegistration)
-                        else -> log.error("Invalid status - should never get here")
+                    val existing =
```

Comment:
same comment as above about not setting status using an actor but assuming success via event stream?